### DISCUSSION TITLE
como usar 32bits miner cpu

### DISCUSSION THREAD

**Discussion by wesleytwe** [revoxhere/duino-coin#716] [General]
ola alguem pode ajudar como miner em 32bts cpu

**Comment by sys-256** [comment]
> ola alguem pode ajudar como miner em 32bts cpu

You can use the download the source code and run it with python.

Você pode usar o download do código-fonte e executá-lo com python. (Eu usei o google tradutor para isso)

**Comment by celoibarros** [comment]
Só seguir as instruções daqui https://github.com/revoxhere/duino-coin.
